util/util.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+import (
+	"fmt"
+	"regexp"
+)
+
+var fullCommandRegexp = regexp.MustCompile(`^([a-zA-Z0-9-_]+)\/([a-zA-Z0-9-_]+)@([a-z0-9-~\*\^\.]+)$`)
+var xrangesRegexp = regexp.MustCompile(`^(?:(\d+)\.)?(?:(\d+)\.)?([\*x]|\d+)$`)
+var tildeRangesRegexp = regexp.MustCompile(`^~\d(\.\d)?(\.\d)?$`)
+var caretRangesAndPinningRegexp = regexp.MustCompile(`^(\^)?\d(\.\d){2}$`)
+var tagRegexp = regexp.MustCompile(`^[a-z0-9-]+$`)
```

Comment:
> Tags must start with a letter and only contain a-z, 0-9, and -. 

Tag should not start with `0-9` and `-`.
https://github.com/screwdriver-cd/data-schema/blob/1375302b10cce8298d55d1b4a0a2e44ed80eb0a8/config/regex.js#L13

Reply:
You are right. I fixed.